test.spec.js

```diff
@@ -0,0 +1,47 @@
+const kijiji = require('./');
+jest.setTimeout(30000);
+
+describe('kijiji-scraper', () => {
+
+    describe('categories', () => {
+
+        it('should return you the kijiji category id for cars and vehicles', () => {
+            expect(kijiji.categories.CARS_AND_VEHICLES.id).toBe(27);
```

Comment:
These category and location tests are just making sure the values are correct, right? I think a more valuable test would be to actually download the current ID numbers from Kijiji and compare against them. I have a script to do this and will check it in during my first pass of writing tests after the PR is merged. These are fine for now.